apps/posterizarr/Dockerfile

```diff
@@ -0,0 +1,48 @@
+FROM docker.io/library/alpine:3.21
```

Comment:
```suggestion
FROM docker.io/library/python:3.13-alpine
```

If you want the notification stuff use this as a base image, then you can run `pip install apprise` in the `RUN`